spec/spec_helper.rb

```diff
@@ -11,6 +11,10 @@
 # Must be explicit as activerecord is optional dependency
 require 'active_record/railtie'
 
+unless File.exist?('config/database.yml')
```

Comment:
@lsanchez-r7 i changed it to a fixed value for now. I think this should not be a real problem because it only affects rspecs